Sources/SwiftkubeClient/Config/KubernetesClientConfig.swift

```diff
@@ -47,7 +51,9 @@ public struct KubernetesClientConfig: Sendable {
 		trustRoots: NIOSSLTrustRoots?,
 		insecureSkipTLSVerify: Bool,
 		timeout: HTTPClient.Configuration.Timeout,
-		redirectConfiguration: HTTPClient.Configuration.RedirectConfiguration
+		redirectConfiguration: HTTPClient.Configuration.RedirectConfiguration,
+		proxyURL: URL? /* = nil*/,
```

Comment:
is it commented out on purpose?

```suggestion
		proxyURL: URL? = nil,
```

Reply:
No, that must have slipped in there somehow. Sorry! 😅